docs/source/model_design/risk_factors.rst

```diff
@@ -48,8 +48,111 @@ Definitions
+
+*Incidence rate* has the same numerator as incidence risk, that is the appearance of new cases. In contrast to risks, which relate the number of new cases to the size of the population at risk in the beginning of the period studied, rates relate the number of new cases to the person-time (Y) at risk, a measure that takes into account changes in the size of the population at risk during the follow-up period. The rate takes into account the fact that some people who start at risk do not remain at risk during the whole period, because they develop the disease, or die, or leave the population by migrating, refusing to continue to participate in the study etc. Others may join the population at risk after the beginning of the period, through birth, migration into the area, recruitment into the study, etc. The denominator in a rate (Y) is thus the sum of the time each person in the study population remained at risk during the study period. This is called the person-time experience at risk, and is expressed in units of person-time: person-years at risk, person-days at risk, baby-weeks at risk etc.
+
+For rare diseases, risk and rates are numerically similar. 
+
+Prevalence
+^^^^^^^^^^
+Prevalence focuses on existing states. Prevalence of a state (such as the 'with condition state') at a point in time may be defined as the proportion of a population in that state at that time; thus prevalence is the proportion of persons in a defined population that have the outcome under study in a defined period of time. 
+
+*Point prevalence* is the number of current cases (new and pre-existing) at a defined instant in time. The denominator is the population at the same defined instant in time. Eg. the percentage of people with schistosomiasis parasites in the blood in a village in Kenya in a survey in December 2019; the proportion of people who have diabetes in China today. 
+
+*Period prevalence* is the number of current cases (new and pre-existing) over a defined period of time. The denominator is the average or mid-period population. 
+
+The *prevalence pool* is the subset of the population who is in the given state (such as the 'with condition state'). A person who dies from the state is removed from the prevalence pool: death decreases prevalence. People can also exit the prevalence pool by recovering from the state (remission) or emigrating from the population. Diseases with high incidence rates may have low prevalence if they are rapidly fatal or quickly cured. Conversely, diseases with low incidence rates may have substantial prevalence if they are nonfatal but incurable. 
+
+Prevalence is seldom of direct interest in etiological applications of epidemiological research because it reflects both incidence rate and duration of disease. However, for congentical disesases, prevalence is the measure usually employed. In our simulations, we call this the birth prevalence. 
+
```

Comment:
Would you include that graph you have, that shows a bunch of cases occurring and remitting over time, and shows incidence vs point prev vs period prev?

Reply:
added!